blockchain/src/blockchain/history_sync.rs

```diff
@@ -378,7 +378,7 @@ impl Blockchain {
         );
 
         this.chain_store
-            .put_chain_info(&mut txn, &block_hash, &chain_info, true);
+            .put_chain_info(&mut txn, &block_hash, &chain_info, false);
```

Comment:
@styppo I believe this can be `false` since we are already storing the `chain_info` with `chain_info.on_main_chain` and we're setting `include_body=true` at line 172.